haiku-apps/passwordgenerator/passwordgenerator-1.0.0.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A simple password generator app"
+DESCRIPTION="This simple password generator app was originally created on pascal." 
```

Comment:
That it was originally a pascal program isn't that important. More interesting would be a short list of its features.